integration_tests/CMakeLists.txt

```diff
@@ -207,7 +207,7 @@ RUN(NAME test_version        LABELS cpython llvm)
 RUN(NAME vec_01              LABELS cpython llvm)
 RUN(NAME test_str_comparison LABELS cpython llvm)
 
-RUN(NAME generics_01         LABELS cpython llvm)
+RUN(NAME generics_01         LABELS cpython)
```

Comment:
You have to keep the `llvm` test here, to ensure it works.

Just not in `run_tests.py`, where it is redundant.

Reply:
Got it